wardenclyffe/mediathread/views.py

```diff
@@ -43,10 +43,8 @@ def mediathread(request):
         return HttpResponse("invalid authentication token")
 
     username = authenticator.username
-    try:
-        user = User.objects.get(username=username)
-    except User.DoesNotExist:
-        user = User.objects.create(username=username)
+    user, created = User.objects.get_or_create(username=username)
+    if created:
```

Comment:
I think this change is good, but I'm not sure it actually changes anything. According to the docs `get_or_create()` is just a convenience method for this try/except pattern, so there's no change in functionality: https://github.com/django/django/blob/master/django/db/models/query.py#L448   https://docs.djangoproject.com/en/1.10/ref/models/querysets/#get-or-create